src/scss/app.scss

```diff
@@ -1,7 +1,9 @@
 @import 'base/normalize';
 @import url('https://fonts.googleapis.com/css2?family=Barlow:wght@600&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Fraunces:wght@700;900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@900&display=swap');
```

Comment:
This fonts imports should be at /base/_typography according to the 7-1 pattern scss
you can check it [here](https://gist.github.com/rveitch/84cea9650092119527bc)